### DISCUSSION TITLE
new enterprise WLANs security modes

### DISCUSSION THREAD

**Discussion by rfl64** [netbox-community/netbox#9388] [Ideas]
WLANs Enterprise security modes like EAP-TLS and so on are not covered by actual version netbox v3.2.2
some fields like authentication method, preshared or certificate options are required
encryption method other than WEP/WPA/WPA2/WPA3 are required

**Comment by DanSheps** [comment]
Are you asking a question or making a statement?  I think we need a little more detail as to what you want before we can contribute to this discussion

**Comment by rfl64** [comment]
is not a question, its an observation about one possible upgrade netbox could have.
